rules/empty-brace-spaces.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+const getDocumentationUrl = require('./utils/get-documentation-url');
+
+const MESSAGE_ID = 'empty-brace-spaces';
+const messages = {
+	[MESSAGE_ID]: 'Do not add spaces between braces.'
+};
+
+const selector = `:matches(${
+	[
+		'BlockStatement[body.length=0]',
+		'ClassBody[body.length=0]',
+		'ObjectExpression[properties.length=0]'
+	].join(', ')
+})`;
+
+const create = context => {
+	const sourceCode = context.getSourceCode();
+	return {
+		[selector](node) {
+			let [start, end] = node.range;
+			start += 1;
+			end -= 1;
+
+			if (!/^\s+$/.test(sourceCode.text.slice(start, end))) {
+				return;
+			}
+
+			context.report({
+				loc: {
+					start: sourceCode.getLocFromIndex(start),
+					end: sourceCode.getLocFromIndex(end)
+				},
+				messageId: MESSAGE_ID,
+				fix: fixer => fixer.replaceTextRange([start, end], '')
+			});
+		}
+	};
+};
+
+const schema = [];
```

Comment:
I don't see the point of adding empty schema.

Reply:
It's in template, forgot to remove.

BTW: Do you know `eslint-plugin-eslint-plugin` not working on our rules? And actually this is [enforced by it](https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/blob/master/docs/rules/require-meta-schema.md).

Reply:
Not sure why it didn't catch this. It has caught issues in the past though.